src/System.Management.Automation/engine/regex.cs

```diff
@@ -403,6 +422,16 @@ private static bool IsWildcardChar(char ch)
             return (ch == '*') || (ch == '?') || (ch == '[') || (ch == ']');
         }
 
+        private static bool IsLeftBracket(char ch)
```

Comment:
Is there a need for these methods, rather than just a simple equality test above?

Reply:
Sure! Removed those methods.